nautobot/core/models/querysets.py

```diff
@@ -102,14 +102,14 @@ def distinct_values_list(self, *fields, flat=False, named=False):
 
     def filter(self, *args, **kwargs):
         """
-        Extend base queryset with support for filtering by `natural_key_slug=...`.
+        Extend base queryset with support for filtering by `composite_key=...`.
 
-        This is an enhanced version of natural-key slug support from django-natural-keys.
-        Counterpart to BaseModel.natural_key_slug property.
+        This is an enhanced version of composite key support from django-natural-keys.
```

Comment:
```suggestion
        This is an enhanced version of natural-key slug key support from django-natural-keys.
```

Since django-natural-keys uses "natural-key slug" instead of "composite key" terminology.